lib/mercury/wire_serializer.rb

```diff
@@ -23,17 +23,15 @@ def write_json(obj)
     end
 
     def read_json(bytes)
-      begin
-        Oj.compat_load(bytes, oj_options)
-      rescue Oj::ParseError => e
-        bytes
-      end
+      Oj.compat_load(bytes, oj_options)
+    rescue Oj::ParseError => e
+      bytes
     end
 
     def oj_options
       {
         mode: :compat,
-        time_format: :xmlschema,  # xmlschema == iso8601
+        time_format: :xmlschema, # xmlschema == iso8601
```

Comment:
We should have freedom to put comments wherever we want and not be space limited.